dot/network/utils_test.go

```diff
@@ -11,6 +11,27 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+const portsAmount = 7100
+
+type portsQueue chan int
+
+func (p *portsQueue) get() int {
+	return <-*p
```

Comment:
no need to have it as a pointer receiver AFAIK since a channel is a pointer itself

Reply:
removed!